File: Tutorials/WinForms/Classic/VB/Form1.vb
Imports DevExpress.Xpo
Imports WinFormsApplication.XpoTutorial

Public Class Form1
    Public Sub New()
        ConnectionHelper.Connect(False)
        Using uow As New UnitOfWork()
            DemoDataHelper.Seed(uow)
        End Using
        InitializeComponent()
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Dim session As New Session
        CustomersBindingSource.DataSource = New XPCollection(Of Customer)(session)
    End Sub
End Class

File: Tutorials/WinForms/Classic/CS/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.Xpo;
using XpoTutorial;

namespace WinFormsApplication {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            Session session = new Session();
            CustomersBindingSource.DataSource = new XPCollection<Customer>(session);
        }
    }
}
